beacon-light-client/plonky2/balance_verification/get_balances_input.ts

```diff
@@ -79,25 +74,25 @@ let TAKE: number;
     ),
   );
 
-  const beaconApi = await getBeaconApi(options['beacon-node']);
+  const beaconApi = await getBeaconApi(config.beaconNodeUrls);
 
   const slot =
-    options['slot'] !== undefined
-      ? BigInt(options['slot'])
+    config.slot !== undefined
+      ? BigInt(config.slot)
       : await beaconApi.getHeadSlot();
   const { beaconState } =
     (await beaconApi.getBeaconState(slot)) ||
     panic('Could not fetch beacon state');
 
   const currentSSZFork = await beaconApi.getCurrentSSZ(slot);
 
-  const offset = Number(options['offset']) || 0;
-  const take = TAKE !== Infinity ? TAKE + offset : Infinity;
+  const offset = Number(config.offset) || 0;
+  const take = config.take !== Infinity ? config.take + offset : Infinity;
   const validators = beaconState.validators.slice(offset, take);
   beaconState.balances = beaconState.balances.slice(offset, take);
   beaconState.validators = validators;
 
-  TAKE = validators.length;
+  config.take = validators.length;
```

Comment:
Why change the config here?
Isn't it more suitable to use a local variable? Making runtime changes on the config seems odd